brightray/browser/notification_presenter.cc

```diff
@@ -23,6 +25,14 @@ base::WeakPtr<Notification> NotificationPresenter::CreateNotification(
   return notification->GetWeakPtr();
 }
 
+void NotificationPresenter::RemoveAllNotifications() {
+  // Copy notifications to avoid iterator invalidation
+  std::vector<Notification*> notifications_to_remove(
+      notifications_.begin(), notifications_.end());
```

Comment:
why not something simple like
`auto notifications_to_remove {notifications_};`?

If this is agreeable, the `#include <vector>` line can also be removed

Reply:
Actually using `auto` with uniform initialization is discouraged in Chromium's coding style:
https://www.chromium.org/developers/coding-style/cpp-dos-and-donts?pli=1#TOC-Variable-initialization